lib/webenginesettings.cpp

```diff
@@ -110,6 +110,8 @@ void SailfishOS::WebEngineSettings::initialize()
     }
     engineSettings->setTileSize(QSize(tileSize, tileSize));
 
+    engineSettings->setPreference(QStringLiteral("browser.viewport.desktopWidth"), QVariant::fromValue(screenWidth));
```

Comment:
This produced some websites with crazy large fonts. I suspect the pixel ratio settings was compensating for what this is trying to fix and the combination blows up in some combinations, https://shop.jolla.com/ for example. A pixel ratio of 1 on a L500D looks about spot on, on an XA2 it's probably a bit squinty.

Reply:
I reduced the default pixel ratio to try and compensate but I'm clearly meddling with forces I don't understand as there's still an excess of scaling on some pages. Scaling the desktopWidth by the pixelRatio didn't seem to help either.

Reply:
Note that this these prefs affects also browser. Do you see difference there when comparing rendering result between browser and WebView?

Reply:
Further, you should not need to adjust desktop viewport value.